packages/cloudflare/src/cli/build/patches/ast/webpack-runtime.ts

```diff
@@ -57,14 +82,26 @@ export async function patchWebpackRuntime(buildOpts: BuildOptions) {
     ".next/server"
   );
 
-  const runtimeFile = join(dotNextServerDir, "webpack-runtime.js");
-  const runtimeCode = readFileSync(runtimeFile, "utf-8");
   // Look for all the chunks.
   const chunks = readdirSync(join(dotNextServerDir, "chunks"))
     .filter((chunk) => /^\d+\.js$/.test(chunk))
     .map((chunk) => {
       return Number(chunk.replace(/\.js$/, ""));
     });
 
-  writeFileSync(runtimeFile, patchCode(runtimeCode, buildInlineChunksRule(chunks)));
+  const runtimeFile = join(dotNextServerDir, "webpack-runtime.js");
+  if (existsSync(runtimeFile)) {
+    let code = readFileSync(runtimeFile, "utf-8");
+    code = patchCode(code, buildMultipleChunksRule(chunks));
+    code = patchCode(code, singleChunkRule);
+    writeFileSync(runtimeFile, code);
+  }
+
+  const apiRuntimeFile = join(dotNextServerDir, "webpack-api-runtime.js");
+  if (existsSync(apiRuntimeFile)) {
+    let code = readFileSync(runtimeFile, "utf-8");
```

Comment:
is runtimeFile instead of apiRuntimeFile here intentional?

Reply:
nope, good catch!